handle-mulipart-form-data/backend_server/server.bal

```diff
@@ -0,0 +1,67 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/random;
+
+public type CustomerRegistrationData record {|
+    string firstName;
+    string lastName;
+    string address;
+    int dependents;
+|};
+
+final table<CustomerRegistrationData> customerTable = table [
```

Comment:
This table should act as a data store. So this table values should hold the `customerId` field as well. Lets define a new `Customer` record and use the `Customer` record to create the table.
```ballerina
type Customer record {|
     * CustomerRegistrationData;
     readonly string customerId;
|};

final table<Customer> key(customerId) customers = table [];
```

Reply:
Yes, but this make sample bit complex ryt?
we only want to show the send and receive capabilities of multi-part data

Reply:
Updated.